pepper-apis/src/main/java/org/broadinstitute/ddp/service/actvityinstancebuilder/ElementCreator.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.ddp.service.actvityinstancebuilder;
+
+
+import org.broadinstitute.ddp.content.Renderable;
+import org.broadinstitute.ddp.model.activity.definition.template.Template;
+
+/**
+ * Abstract creator class
+ */
+public class ElementCreator {
```

Comment:
Is there advantage to having this `ElementCreator`? I don't see how having this parent class helps us. If we really need these, I think `getTemplateId()` and `applyRenderedTemplates()` could be implemented as methods on the `Context` class itself.